### PR TITLE
simple kmeans

### PR DESCRIPTION
**Pull request by zhelinliao**

### CODE REVIEW

File: clustering/kmeans.py [hunk 1]
```diff
@@ -1,17 +1,25 @@
...
-
 def point_avg(points):
     """
     Accepts a list of points, each with the same number of dimensions.
     (points can have more dimensions than 2)
     
     Returns a new point which is the center of all the points.
     """
-    raise NotImplementedError()
+    new_center[]
```
Comment (gallettilance) [inline clustering/kmeans.py]: Not sure this is correct python3 syntax

Comment (zhelinliao) [reply clustering/kmeans.py]: I've pushed an update on the code. Thank you professor.

**Comment by gallettilance** [comment]
@zhelinliao Looks like there are still some errors. Can you take a look at the test output and debug accordingly? Let me know if you have any questions about how to test locally.

**Comment by zhelinliao** [comment]
> @zhelinliao Looks like there are still some errors. Can you take a look at the test output and debug accordingly? Let me know if you have any questions about how to test locally.

**Pull request closed without merging**